pkg/metrics/metrics_test.go

```diff
@@ -707,3 +708,20 @@ func containsMetrics(expectedMfs, gotMfs []*cmg.MetricFamily) bool {
 
 	return true
 }
+
+func TestProcessStartTimeMetricExist(t *testing.T) {
+	mgr, wg, srv := initMgr()
+	defer shutdown(srv, wg)
+	metricsFamilies, err := mgr.GetRegistry().Gather()
+	if err != nil {
+		t.Fatalf("Error fetching metrics: %v", err)
+	}
+
+	for _, metricsFamily := range metricsFamilies {
+		if metricsFamily.GetName() == processStartTimeMetric {
```

Comment:
nit: can we check for a non-zero metric value for `metricsFamily`? To make sure the processes start time is registered.

Reply:
Done